src/main/webapp/partials/instructorHelpQuestions.jsp

```diff
@@ -358,589 +358,1614 @@
+                    </td>
+                    <td>
+                      <input class="visibilityCheckbox answerCheckbox" type="checkbox" value="INSTRUCTORS" disabled="" checked="">
+                    </td>
+                    <td>
+                      <input class="visibilityCheckbox giverCheckbox" type="checkbox" value="INSTRUCTORS" disabled="" checked="">
+                    </td>
+                    <td>
+                      <input class="visibilityCheckbox recipientCheckbox" type="checkbox" value="INSTRUCTORS" disabled="" checked="">
+                    </td>
+                  </tr>
+                </tbody></table>
+              </div>
+              <!-- Fix for collapsing margin problem. Reference: http://stackoverflow.com/questions/6204670 -->
+              <div class="visibility-message overflow-hidden" id="visibilityMessage-1">
+                This is the visibility hint as seen by the feedback giver:
+                <ul class="text-muted background-color-warning">
+                  <li>You can see your own feedback in the results page later on.</li>
+                  <li>Instructors in this course can see your response, the name of the recipient, and your name.</li>
+                  </ul>
+              </div>
+            </div>
+            <div>
+              <span class="pull-right">
+              <input id="button_question_submit-1" type="submit" class="btn btn-primary" value="Save Changes" tabindex="0" style="display:none" disabled="">
+              </span>
+            </div>
                 </div>
               </div>
+              <input type="hidden" name="fsname" value="MCQ weights">
+              <input type="hidden" name="courseid" value="CS3424">
+              <input type="hidden" name="questionid" value="ahF0ZWFtbWF0ZXMtc3VrYW50YXIdCxIQRmVlZGJhY2tRdWVzdGlvbhiAgICAgKTLCQw">
+              <input type="hidden" name="questionnum" value="1">
+              <input type="hidden" name="questiontype" value="MCQ">
+              <input type="hidden" name="questionedittype" id="questionedittype-1" value="edit">
+              <input type="hidden" name="showresponsesto" value="RECEIVER,INSTRUCTORS">
+              <input type="hidden" name="showgiverto" value="RECEIVER,INSTRUCTORS">
+              <input type="hidden" name="showrecipientto" value="RECEIVER,INSTRUCTORS">
+              <input type="hidden" name="user" value="test@example.com">
+              <input type="hidden" name="token" value="95DAF259F7FEFD63C0D3AD723C82437B">
             </form>
           </div>
           <p>
             When you view the results of a multiple-choice (single answer) question, TEAMMATES calculates some statistics about the results collected,
             such as the number of responses for each option, and the percentage of response in which each option was chosen.
+            If weights are assigned in the question, then additional statistics are calculated. Such as, average point for each option.
```

Comment:
`... additional statistics are calculated such as count and average point for each option.`